components/organisms/SignUpForm/SignUpForm.js

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react';
+
+import SignUpFormContent from './SignUpFormContent';
+import useSignUp from './useSignUp';
+
+const SignUpForm = ({ onSuccess }) => {
+  const {
+    signUp,
+    mutationResult: { error, loading },
+  } = useSignUp({ onSuccess });
```

Comment:
I think this is not a good solution to pass callbacks into mutation hooks. In my opinion, this makes difficult to understand why it always should have callback. How about to define a submit handler inside component like that:

```
const onSubmit = async data => {
  try {
     await signUp(data);
     await onSuccess():
  } catch () {
  ....
 }
}
```